mongodb/symfony-bundle/0.1/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "MongoDB\\Bundle\\MongoDBBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "MONGODB_URI": ""
```

Comment:
Can we set a default value to show the format?
```suggestion
        "MONGODB_URI": "mongodb://localhost:27017/"
```

Reply:
why not 👍